src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,25 @@
+.calendar {
```

Comment:
place the calendar at the center of the page to avoid hiding days when they're hovered
<img width="928" alt="Screenshot 2023-07-20 at 20 20 25" src="https://github.com/mate-academy/layout_calendar/assets/117587463/63371961-4a99-4202-bf32-95da8d77e070">